py/autoevals/oai.py

```diff
@@ -5,93 +5,146 @@
 import time
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Any
+from typing import Any, Optional
 
 PROXY_URL = "https://api.braintrust.dev/v1/proxy"
 
 
 @dataclass
-class OpenAIWrapper:
+class AutoEvalClient:
```

Comment:
maybe name it `LLMClient`? Or I guess it extends beyond LLMs theoretically?

Reply:
yeah let's go with that 👍🏻